### ISSUE TITLE
First time ever `npm install cypress` never finishes

### ISSUE THREAD

**Issue by alexsch01** [cypress-io/cypress#30779]
### Current behavior

First time running the command `npm install cypress`, it never completes
When I run it the second time, it will finish

### Test code to reproduce

Doesn't apply to the issue

### Cypress Version

13.17.0

### Node version

v22.11.0

### Operating System

Fresh Fedora 41 toolbox with nodejs

### Other

Not a regression

**Comment by jennifer-shehane** [comment]
@alexsch01 Is this specific to 13.17.0?

Can you provide DEBUG logs?

**Comment by alexsch01** [comment]
Nope

**Comment by jennifer-shehane** [comment]
I wasn't able to reproduce this on a fresh directory. We'd need some more information to debug this.

**Comment by alexsch01** [comment]
Probably a npm issue since I don't see any debug information with `export DEBUG=*` run before

**Comment by alexsch01** [comment]
npm 10.9.0 has this issue
It was fixed in npm 10.9.1

**Comment by MikeMcC399** [comment]
If the issue repeats itself, then refer to the troubleshooting steps on https://docs.cypress.io/app/references/advanced-installation#Troubleshoot-installation which list the following instructions to collect debug logs:

```shell
CYPRESS_INSTALL_BINARY=0 npm install cypress --save-dev
DEBUG=cypress:cli* npx cypress install
```

**Comment by alexsch01** [comment]
@MikeMcC399 yeah I believe it was a npm specific issue since none of the DEBUG logs were showing

**Comment by MikeMcC399** [comment]
@alexsch01 

>  yeah I believe it was a npm specific issue since none of the DEBUG logs were showing

I was unable to reproduce your problem on Fedora 41 with your version of Node / npm. Without logs it won't be possible to diagnose. If it is all working now then there is no need to dig deeper.

**Comment by alexsch01** [comment]
Removing $HOME/.npm folder and removing node_modules directory in project folder
npm install cypress (using npm 10.9.0) gives this issue 

npm 10.9.1 doesn't for me

**Comment by MikeMcC399** [comment]
@alexsch01 

Executing the following under Fedora 41, Node.js `v22.11.0`, npm `10.9.0` is successful, so this is not a general problem:

```shell
rm -rf ~/.npm
rm -rf ~/.cache/Cypress
rm -rf node_modules
CYPRESS_INSTALL_BINARY=0 npm install cypress --save-dev
DEBUG=cypress:cli* npx cypress install
```

## Logs

```text
$ CYPRESS_INSTALL_BINARY=0 npm install cypress --save-dev
DEBUG=cypress:cli* npx cypress install

added 172 packages, and audited 173 packages in 7s

39 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
npm notice
npm notice New major version of npm available! 10.9.0 -> 11.0.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v11.0.0
npm notice To update run: npm install -g npm@11.0.0
npm notice
  cypress:cli:cli cli starts with arguments ["/home/mike/n/bin/node","/home/mike/github/cy-test/node_modules/.bin/cypress","install"] +0ms
  cypress:cli NODE_OPTIONS is not set +0ms
  cypress:cli:cli program parsing arguments +6ms
  cypress:cli parsed cli options {} +167ms
  cypress:cli installing with options {} +0ms
  cypress:cli detecting arch { osPlatform: 'linux', osArch: 'x64' } +5ms
  cypress:cli arm uname -m result: { stdout: 'x86_64' }  +63ms
  cypress:cli version in package.json is 13.17.0, version to install is 13.17.0 +64ms
  cypress:cli Reading binary package.json from: /home/mike/.cache/Cypress/13.17.0/Cypress/resources/app/package.json +0ms
  cypress:cli no binary installed under cli version +22ms
  cypress:cli checking local file /home/mike/13.17.0 cwd /home/mike/github/cy-test +0ms
  cypress:cli preparing to download and unzip version  13.17.0 to path /home/mike/.cache/Cypress/13.17.0 +0ms
Installing Cypress (version: 13.17.0)


  cypress:cli needed Cypress version: 13.17.0 +0ms
  cypress:cli source url https://download.cypress.io/desktop/13.17.0?platform=linux&arch=x64 +0ms
  cypress:cli downloading cypress.zip to "/tmp/cypress-3430.zip" +0ms
  cypress:cli Downloading package {
  url: 'https://download.cypress.io/desktop/13.17.0?platform=linux&arch=x64',
  proxy: null,
  downloadDestination: '/tmp/cypress-3430.zip'
} +1ms
  cypress:cli expected file size 82 +484ms
  cypress:cli redirect version: 13.17.0 +2ms
  cypress:cli redirect url: https://cdn.cypress.io/desktop/13.17.0/linux-x64/cypress.zip +0ms
  cypress:cli Downloading package {
  url: 'https://cdn.cypress.io/desktop/13.17.0/linux-x64/cypress.zip',
  proxy: null,
  downloadDestination: '/tmp/cypress-3430.zip'
} +0ms
  cypress:cli expected checksum 74e5d06f8569ff3177d89f0b04cec3ae0102d80cdebb3931d42c4d86772c00ac3fcfbc35981283b1a414681ed6ee334e51f6e42ef1b777553b34fa8ab8825358 +118ms
⠙  Downloading Cypress      100% 0s
  cypress:cli downloading finished +36s
⠦  Downloading Cypress      100% 0s
  cypress:cli downloaded file has the expected checksum and size ✅ +513ms
  cypress:cli verified +0ms
✔  Downloaded Cypress
  cypress:cli:unzip unzipping from /tmp/cypress-3430.zip +0ms
  cypress:cli:unzip into /home/mike/.cache/Cypress/13.17.0 +0ms
  cypress:cli:unzip zipFile entries count 25513 +5ms
✔  Downloaded Cypress
✔  Downloaded Cypress
✔  Downloaded Cypress
✔  Unzipped Cypress
✔  Downloaded Cypress
✔  Unzipped Cypress
✔  Finished Installation /home/mike/.cache/Cypress/13.17.0

You can now open Cypress by running one of the following, depending on your package manager:

- npx cypress open
- yarn cypress open
- pnpm cypress open

https://on.cypress.io/opening-the-app
```
